### PR TITLE
Add getConnectionString() for MongoDBContainer

### PR DESCRIPTION
**Pull request by mdedetrich**
The Mongodb module currently does not have a helper method for getting the raw connection string without pointing to a database (there is `getReplicaSetUrl` but that always points to a database). Some tests work with their own database that aren't named `test` (i.e. https://github.com/akka/alpakka/pull/2857). Furthermore (and as can be seen in the referenced PR) `MONGODB_INTERNAL_PORT` is private so creating your own `getReplicaSetUrl` has even more boilerplate.

I am not sure if there is a better name than `getReplicaSetUrl`, if not let me know.

### CODE REVIEW

**Comment by kiview** [comment]
Something seems to be wrong with the Javadoc, can you please check this locally? Also, could you add a test that uses the new method?

File: modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java [hunk 1]
```diff
@@ -48,6 +48,19 @@ public MongoDBContainer(final DockerImageName dockerImageName) {
...
 
+    /**
+     * Gets a connection string url, unlike {@value #getReplicaSetUrl} this does point to a
+     * database
+     * @return a connection url pointing to a mongodb instance
+     */
+    public String getConnectionString() {
+        return String.format(
+            "mongodb://%s:%d",
+            getContainerIpAddress(),
```
Comment (eddumelendez) [inline modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java:59]:
```suggestion
            getHost(),
```

File: modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java [hunk 2]
```diff
@@ -48,6 +48,19 @@ public MongoDBContainer(final DockerImageName dockerImageName) {
         );
     }
 
+    /**
+     * Gets a connection string url, unlike {@value #getReplicaSetUrl} this does point to a
```
Comment (eddumelendez) [inline modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java:52]:
```suggestion
     * Gets a connection string url, unlike {@link #getReplicaSetUrl} this does point to a
```

**Review by eddumelendez — COMMENTED** [review]
the javadoc should be `link` instead of `value`. After that the checks should pass.

**Comment by mdedetrich** [comment]
@kiview I will write a test tomorrow

**Review by kiview — APPROVED** [review]

**Review by bsideup — APPROVED** [review]

**Pull request merged**